src/gtimelog/ui/components/utils.py

```diff
@@ -0,0 +1,138 @@
+"""
+UI components utils
+"""
+import os
+import sys
+import collections
+import smtplib
+from gettext import gettext as _
+import gi
+from gi.repository import Gtk, GLib, GObject, Gio, Gdk
+
+from gtimelog import root_logger, __version__
+from gtimelog.core.settings import Settings
+from gtimelog.core.utils import as_minutes
+from gtimelog.core.reports import ReportRecord
+
+log = root_logger.getChild('application')
```

Comment:
I probably already mentioned that I don't like this.  `log = logging.getLogger(__name__)` is fine.

Reply:
Fixed